src/bogart/AS_BAT_MarkRepeatReads.C

```diff
@@ -1009,7 +879,7 @@ findConfusedEdges(TigVector            &tigs,
                    external3sco.tigId, external3sco.readId,
                    internal3sco.score, external3sco.score, ad3, pd3);
 
-          confusedEdges.push_back(confusedEdge(rdAid, false, external3sco.readId));
+          confusedEdges.push_back(confusedEdge(rdAid, true, external3sco.readId));
```

Comment:
What has changed?

Reply:
This was adding a confused edge to the 5' end of a read when it should have been the 3' end.  These are used to make unitigs.